package.json

```diff
@@ -122,6 +125,7 @@
     "swagger-tools": "^0.10.1",
     "throng": "^1.0.1",
     "underscore": "^1.8.3",
+    "url": "^0.11.0",
```

Comment:
you should only need to add this for client-side use... when running within node.js, the core node.js "url" module is always available from a require('url') without adding it here